ConfigurationSystem/Client/Helpers/Resources.py

```diff
@@ -114,36 +118,6 @@ def getSiteGrid( site ):
     return S_ERROR( 'Wrong Site Name format' )
   return S_OK( sitetuple[0] )
 
-def getStorageElementOptions( seName ):
```

Comment:
Don't you want to deprecate that first ? Maybe an extension is using it ? (I highly doubt, but... well)

Reply:
OK done.